Radzen.Blazor/wwwroot/Radzen.Blazor.js

```diff
@@ -1699,7 +1699,15 @@ window.Radzen = {
           },
           mouseMoveHandler: function (e) {
               if (Radzen[el]) {
-                  var width = (Radzen[el].width - (Radzen[el].clientX - e.clientX)) + 'px';
+                  var widhtFloat = (Radzen[el].width - (Radzen[el].clientX - e.clientX));
```

Comment:
Just a small typo - should be `width`

Reply:
fixed it in my last commit. thanks for the review.